include/readout/concepts/RequestHandlerConcept.hpp

```diff
@@ -33,10 +35,11 @@ class RequestHandlerConcept
   virtual void start(const nlohmann::json& args) = 0;
   virtual void stop(const nlohmann::json& args) = 0;
   virtual void record(const nlohmann::json& args) = 0;
+  virtual void get_info(datalinkhandlerinfo::Info&) = 0;
 
   // requests
-  virtual void auto_cleanup_check() = 0;
-  virtual void issue_request(dfmessages::DataRequest /*dr*/, unsigned /*delay_us*/ = 0) = 0;
+  virtual void cleanup_check() = 0;
```

Comment:
Doxy comments: optional cleanup procedure interface. Same for the funciton below.